packages/@o3r/eslint-plugin/src/index.ts

```diff
@@ -39,19 +39,34 @@ module.exports = {
       }
     },
 
+    // deprecated: should use `angular-template-recommended` instead. Will be removed in v13.
     'template-recommended': {
       rules: {
         '@o3r/no-folder-import-for-module': 'error',
         '@o3r/template-async-number-limitation': 'warn'
       }
     },
 
+    // deprecated: should use `monorepo-recommended` instead. Will be removed in v13.
     'json-recommended': {
       rules: {
         '@o3r/json-dependency-versions-harmonize': 'error'
       }
     },
 
+    'angular-template-recommended': {
+      rules: {
+        '@o3r/no-inner-html': 'off',
```

Comment:
It is the current one.
It can be changed, in the big eslint refactor if we want

Reply:
We don't want to prevent the usage of innerHTML, this rule as been done for project that prefer innerText 
In fact we should rename that rule to `@o3r/prefer-inner-text` could be done on the next major but we will still keep this rule out of recommended